lib/tasks/export_existing_firms_without_accounts.rake

```diff
@@ -0,0 +1,6 @@
+namespace :export do
```

Comment:
Namespaces probably should be nouns rather than verbs, and the task name is a little confusing for me.

How about `principals:generate_invitations` — or something